manim/cli/checkhealth/checks.py

```diff
@@ -154,7 +155,10 @@ def is_ffmpeg_working():
     ),
 )
 def is_latex_available():
-    path_to_latex = shutil.which("latex")
+    if config.get_dir("tex_bin_dir") is not None:
+        path_to_latex = config.get_dir("tex_bin_dir") / "latex.exe"
```

Comment:
I've tested this on Windows, and found the ".exe" suffix to be necessary. This might be different in other OS's though, in which case a simple if clause should solve it.